website/eacr/templates/menus/footer_menu.html

```diff
@@ -0,0 +1,12 @@
+{% load menu_tags i18n %}
+{% load static %}
+
+<nav class="navbar navbar-toggleable-md navbar-left">
+    <ul class="navbar-nav col-md-12  main-menu">
+        {% for item in menu_items %}
+            <li class="nav-item {{ item.active_class }}">
+                <a class="nav-link" style="color: grey;" href="{{ item.href }}">{{ item.text }}</a>
```

Comment:
Yeah, it is a hack... :-1: I couldn't find out why my site specific custom style sheet is overriden by boostrap `_reboot` or what. I am very curious to find out why, but already lost to much time on this nonsense.